plugins/catalog-react/src/filters.ts

```diff
@@ -108,6 +108,16 @@ export class EntityTextFilter implements EntityFilter {
     return true;
   }
 
+  getBackendRequestParameter() {
```

Comment:
maybe?

```suggestion
  getFullTextFilters() {
```

Reply:
I feel like I want this to be a top level method, could use it to change the sort order as well and `getFullTextFilters` would just be a oneoff.

Reply:
But then you lose typings, meaning that if you make it generic, the method would need to return a generic object, which makes it easier to make mistakes

Reply:
You've convinced me 😁, updated in [bc42ec3](https://github.com/backstage/backstage/pull/22131/commits/bc42ec30b5c8feb20d5c4789cc561509ae649cfb). This should be ready for another round.